src/main/java/org/jetbrains/research/anticopypaster/ide/RefactoringNotificationTask.java

```diff
@@ -43,14 +44,20 @@ public class RefactoringNotificationTask extends TimerTask {
             .getNotificationGroup("Extract Method suggestion");
     private final Timer timer;
     private PredictionModel model;
+    private boolean didIt;
```

Comment:
minor nitpick, could we change this variable name to something a bit more descriptive of what's being done? didIt is pretty vague when reading through with no context

Reply:
I like this new way of doing it much more. Manually tested and pushed